resources/configure_dependencies.py

```diff
@@ -138,7 +152,7 @@ def ExecSub(command: str,
 make_command = "make"
 s, e, outstr = ExecSub(make_command + " --version")
 if outstr.find("GNU Make") >= 0:
-    make_command = "make OMAKE_PRINTDIR=gmake"
+    make_command = "make OMAKE_PRINTDIR=make"
```

Comment:
can you remind me what the issue was between make and gmake here? Thanks

Reply:
There is no issue. This fix came from what PETSc support sent me long ago when I was unable to build PETSc. Theirs used `gmake` for both, I have always used `make` for both and it works fine. This was just done for consistency.